src/v/storage/tests/appender_chunk_manipulations.cc

```diff
@@ -53,7 +53,7 @@ SEASTAR_THREAD_TEST_CASE(chunk_manipulation) {
         // 10 bytes on the next page
         BOOST_REQUIRE_EQUAL(c.flushed_pos() % alignment, 10);
         BOOST_TEST_MESSAGE("10 bytes spill over: " << c);
-        c.append(b.data() + i, alignment + 10);
+        c.append(b.data() + i, static_cast<size_t>(alignment) + 10);
```

Comment:
I'm struggling with this type as `size_t`. I don't really know why. Maybe `alignment()` would make me feel easier. I think it's the `const char*` on the left.

Reply:
i guess the implicit conversion to size_t would invoke `operator type()` on the named_type to get the value. it looks weird cause we know its a class? seems ok tho.

Reply:
While I was rebasing this I changed it to an `alignement()`